internal/handler/msg_serverstatus.go

```diff
@@ -69,6 +70,41 @@ func (h *Handler) msgServerStatus(connCtx context.Context, req *middleware.Reque
 		}
 	}
 
+	// Add CommandFeatureCounterStats as Prometheus metrics
```

Comment:
This comment doesn't seem correct, there is no Prometheus metrics here. I don't think comment like this are necessary :hugs: